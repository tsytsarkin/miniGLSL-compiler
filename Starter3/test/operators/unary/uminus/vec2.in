{ vec2 x; y = -x; }
