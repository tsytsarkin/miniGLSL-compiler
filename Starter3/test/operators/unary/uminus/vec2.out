  1: { vec2 x; y = -x; }
(SCOPE (DECLARATIONS (DECLARATION x vec2 ) ) (STATEMENTS (ASSIGN unknown y (UNARY vec2 - x ) ) ) ) 
Failed to compile
